open-source-services.md

```diff
@@ -28,19 +28,47 @@ Thus we must address common questions about the requirements:
 
 ### Q: What defines a service?
 
-A service is software operated for the user. Software the user is operating for themself is not considered a service. A service is defined by the context in which the software is operated and all of the assets required to operate the software for the user in that context.
+A service is software operated for the user.
+Software the user is operating for themself is not considered a service.
+A service is defined by the context in which the software is operated and all of the assets required to operate the software for the user in that context.
 
-### Q: What “assets” should be Open Source?
+This is a lightweight definition of service, read the Red Hat ["What are cloud services?"](https://www.redhat.com/en/topics/cloud-computing/what-are-cloud-services) article for a more broad discussion of cloud services.
+
+### Q: What defines "Open Source"?
+
+Where the [Open Source definition](https://opensource.org/osd) tells us how to identify a software code base as being Open Source, that definition does not cover what is needed to Open a service.
+
+For that we use the [Open Definition 2.1](https://opendefinition.org/od/2.1/en/), which starts with this summary of the definition:
+
+> _Knowledge is [O]pen if anyone is free to access, use, modify, and share it — subject, at most, to measures that preserve provenance and openness._
+
+For an Open Source code base to run as an Open Source service, there are a number of [Open Works](https://fossrit.github.io/open-work-definition/) around it needed to make it run in a cloud or service environment: scripts, documentation, policies, configurations, and so forth.
+
+### Q: What "assets and dependencies" should be Open Source?
```

Comment:
Why was the word "dependencies" added here? This question is not about dependencies, rather the one below is.